include/oneapi/dpl/pstl/tuple_impl.h

```diff
@@ -37,6 +37,23 @@ struct tuple;
 
 namespace std
 {
+//forward declare new std::get<I>() functions so they can be used in impl
```

Comment:
Move these declarations down, below the specializations of `tuple_element` etc.? They use `std::tuple_element_t`, and perhaps it is even OK for forward declarations but why not avoid this concern.

Reply:
done, thanks.